### PR TITLE
Remove unneeded fields from config files

### PR DESCRIPTION
**Pull request by VIKTORVAV99**
This PR brings over some of the changes I made to the production app a while ago.
It reduces the config chunk from 277.10 KiB (gzip: 39.71 KiB) to just 90.84 KiB (gzip: 24.45 KiB) without affecting any of our current functionality.

Should bring up the performance score just a little as well due to the reduced size, loading time and reduced parsing time.

Related issue: https://linear.app/electricitymaps/issue/ELE-1525/app-improve-bundle-size

### CODE REVIEW

**Comment by VIKTORVAV99** [comment]
If we re-work it a little we could simplify the following check as well and reduce the size even further by merging the parser and estimation into one label/check.
https://github.com/electricitymaps/electricitymaps-contrib-rewrite/blob/13f20277587438ff5100f3ad5cef30a143ddb9ea/web/src/features/panels/zone/util.ts#LL45-L52C4

**Comment by VIKTORVAV99** [comment]
@madsnedergaard do you think we should combine these two fields in the build step and remove the check in the frontend (and just assign the value) or would that have unintended side effects?

File: web/scripts/generate-zones-config.ts [hunk 1]
```diff
@@ -42,11 +55,17 @@ const mergeExchanges = () => {
   const exchangeFiles = fs.readdirSync(basePath);
   const filesWithDirectory = exchangeFiles.map((file) => `${basePath}/${file}`);
 
+    const UNNECESSARY_EXCHANGE_FIELDS = ['capacity', 'comment', '_comment', 'parsers'];
```
Comment (tonypls) [inline web/scripts/generate-zones-config.ts:58]: We do need exchange capacity for the breakdown chart

Comment (VIKTORVAV99) [reply web/scripts/generate-zones-config.ts:58]: Unless you guys changed things we get the capacity from the backend no?

**Comment by madsnedergaard** [comment]
> @madsnedergaard do you think we should combine these two fields in the build step and remove the check in the frontend (and just assign the value) or would that have unintended side effects?

It would generally be nice to move parts of the "data status" state management to app-backend, but I think it would be nice to address on a larger scale and rework how/what is available where and when :)

File: web/scripts/generate-zones-config.ts (hunk 1 above)
Comment (madsnedergaard) [reply web/scripts/generate-zones-config.ts:58]: It currently doesn't show capacity bars, so something have definitely changed somewhere :)

Comment (VIKTORVAV99) [reply web/scripts/generate-zones-config.ts:58]: Oh okay. I'd happily add back the exchanges capacities for now but I do think we should change back the behavior fetching these from the backend, not only to align it with how we fetch the production data but also in preparation for any future capacity parsers. The last part might be especially important for exchanges as they have both dynamic NTC and maximum NTCs.

I'd happily look into it but TBH I'm not too sure how we do things after you guys changed it all 😅

**Comment by VIKTORVAV99** [comment]
I originally intended this to be a separate PR but to avoid merge conflicts I added it here:
But with rounding the config chunk are now: 82.33 KiB (gzip: 19.24 KiB) and the world chunk 712.26 KiB (gzip: 250.15 KiB)

**Comment by VIKTORVAV99** [comment]
Marking this as ready for review now so we can get it merged then we can look into the exchange capacities in another PR.

**Comment by VIKTORVAV99** [comment]
I noticed there are plenty of other fields we could potentially remove as well such as `flag_file_name`, `subZoneNames` and `bypassedSubZones`. Should I include them as well?

**Comment by madsnedergaard** [comment]
> I noticed there are plenty of other fields we could potentially remove as well such as `flag_file_name`, `subZoneNames` and `bypassedSubZones`. Should I include them as well?

- `flag_file_name` should probably just be removed from all zone configs directly instead :) 
- `subZoneNames` seems to still be used (in geo/generate-aggregates.ts)
- `bypassedSubZones` could be removed

**Comment by madsnedergaard** [comment]
Given the fairly limited impact of adding those though, I don't think you should spend too much time on it :)

**Review by madsnedergaard — APPROVED** [review]

**Comment by VIKTORVAV99** [comment]
They cut out about 8kb, there where a lot of those flag names 😅
Anyway it's not the size improvements I'm looking for it's the parsing ones as we are spending a lot of time just parsing our JS files.

**Comment by VIKTORVAV99** [comment]
@madsnedergaard I have no merge powers in here 😅

**Comment by VIKTORVAV99** [comment]
> > @madsnedergaard I have no merge powers in here 😅
> 
> Oh yeah, keep forgetting about that 😬 I'll set it up when I'm at the laptop

Let's just merge this to master instead 😉

**Pull request merged**